requirements.txt

```diff
@@ -3,6 +3,7 @@ certifi==2022.12.7
 charset-normalizer==3.1.0
 decorator==5.1.1
 idna==3.4
-requests==2.28.2
+pyyaml==6.0
+requests==2.29.0
```

Comment:
this versions is not matching the one in the Pipfile

Reply:
I'm not sure what you mean, this matches Pipfile.lock?

Reply:
Not Pipfile.lock, I see this `requests = "~=2.28"` in the Pipfile

Reply:
the `~=` version constraint specifier comes from [PEP440](https://peps.python.org/pep-0440/#compatible-release) and will treat the least significant version portion as a `*`, so `~=2.28` is equivalent to `==2.*`. so when I added pyyaml to the Pipfile and ran `pipenv update ; pipenv requirements --exclude-markers > requirements.txt`, pipenv updated requests to the best available matching version: `2.29.0`